server/service/src/sync/translations/requisition_line.rs

```diff
@@ -16,10 +16,9 @@ use super::{PullTranslateResult, PushTranslateResult, SyncTranslation};
 
 #[derive(Deserialize, Serialize, PartialEq)]
 pub struct RequisitionLineOmsFields {
-    #[serde(deserialize_with = "empty_str_as_option_string")]
-    pub rnr_form_line_id: Option<String>,
-    #[serde(deserialize_with = "empty_str_as_option")]
```

Comment:
I'm quite certain these annotations are not needed for properties of objects coming from OG, just cargo cult from the legacy record translation. OG's more primitive field types will set null to empty values (e.g. null String -> "", null number -> 0, bool -> false...). This is not true of object field's content other than null Object -> empty Object `{}`. Beyond that there is no schema.